gluten-core/src/main/scala/io/glutenproject/backendsapi/SparkPlanExecApi.scala

```diff
@@ -199,6 +199,10 @@ trait SparkPlanExecApi {
     throw new UnsupportedOperationException("NaNvl is not supported")
   }
 
+  def genUuidTransformer(substraitExprName: String, original: Uuid): ExpressionTransformer = {
+    GenericExpressionTransformer(substraitExprName, Seq(), original)
```

Comment:
Could you remind me why we distinguish the implementations of Velox and CH backend?

Reply:
Uuid is a leaf expression, looks CH just treat it as leaf expression and ignore the seed gerenated by spark.
Velox uuid can accept the seed parameter as a constant input. We get the seed and pass it to velox. This is the reason we distinguish the implements.
Thanks.

Reply:
cc: @zzcclp